pkg/static/login.js

```diff
@@ -922,6 +972,15 @@
         let wanted = window.sessionStorage.getItem('login-wanted');
         const machine = id("server-field").value;
 
+        /* When using cockpit client remember all the addresses being used */
+        if (machine && environment.is_cockpit_client) {
+            const hosts = JSON.parse(localStorage.getItem("cockpit-client-sessions") || "[]");
```

Comment:
...and this one too :/

Maybe there ought to be a function?

Reply:
oops, fixed